src/Annotation/SmsGateway.php

```diff
@@ -50,4 +50,13 @@ class SmsGateway extends Plugin {
    */
   protected $schedule_aware;
 
+  /**
+   * Whether the gateway supports queries of current credit balance.
+   *
+   * @see \Drupal\sms\Entity\SmsGatewayInterface::supportsCreditBalanceQuery()
+   *
+   * @var boolean
+   */
+  protected $credit_balance_query;
```

Comment:
Warning: Another of my variable naming nitpicks.

A more descriptive name would be `$credit_balance_available`.